### PR TITLE
Add reuse guidelines attachment rewrite rules

### PR DESCRIPTION
**Pull request by libremente**
This PR:
* adds some rewrite rules to cope with https://github.com/italia/lg-acquisizione-e-riuso-software-per-pa-docs/pull/52.

Should we use some "smarter" rules or the complete rewrite may be good?
@alranel please review, thanks!

### CODE REVIEW

File: roles/app/templates/nginx/snippets/docs-redirects.conf [hunk 1]
```diff
@@ -2,3 +2,10 @@
 # This is to prevent broken links pointing to version "stabile"
 # The version "stabile" needs to be deactivated
 rewrite ^/italia/piano-triennale-ict/pianotriennale-ict-doc/it/stabile/(.*)$ /italia/piano-triennale-ict/pianotriennale-ict-doc/it/2017-2019/$1 permanent;
+
+# Rewrite rules for reuse guidelines
+rewrite ^/allegato-b-guida-alla-pubblicazione-open-source-di-software-realizzato-per-la-pa.html$ /allegato-a-guida-alla-pubblicazione-open-source-di-software-realizzato-per-la-pa.html permanent;
```
Comment (pdavide) [inline roles/app/templates/nginx/snippets/docs-redirects.conf]:
```suggestion
rewrite ^/italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/(.+)/attachments/allegato-b-guida-alla-pubblicazione-open-source-di-software-realizzato-per-la-pa.html$ /italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/$1/attachments/allegato-a-guida-alla-pubblicazione-open-source-di-software-realizzato-per-la-pa.html permanent;
```

File: roles/app/templates/nginx/snippets/docs-redirects.conf [hunk 2]
```diff
@@ -2,3 +2,10 @@
...
+rewrite ^/allegato-c-guida-alla-manutenzione-di-software-open-source.html$ /allegato-b-guida-alla-manutenzione-di-software-open-source.html permanent;
```
Comment (pdavide) [inline roles/app/templates/nginx/snippets/docs-redirects.conf]:
```suggestion
rewrite ^/italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/(.+)/attachments/allegato-c-guida-alla-manutenzione-di-software-open-source.html$ /italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/$1/attachments/allegato-b-guida-alla-manutenzione-di-software-open-source.html permanent;
```

File: roles/app/templates/nginx/snippets/docs-redirects.conf [hunk 3]
```diff
@@ -2,3 +2,10 @@
...
+rewrite ^/allegato-d-guida-alle-licenze-open-source.html$ /allegato-c-guida-alle-licenze-open-source.html permanent;
```
Comment (pdavide) [inline roles/app/templates/nginx/snippets/docs-redirects.conf]:
```suggestion
rewrite ^/italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/(.+)/attachments/allegato-d-guida-alle-licenze-open-source.html$ /italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/$1/attachments/allegato-c-guida-alle-licenze-open-source.html permanent;
```

File: roles/app/templates/nginx/snippets/docs-redirects.conf [hunk 4]
```diff
@@ -2,3 +2,10 @@
...
+rewrite ^/allegato-e-guida-alla-modifica-di-software-open-source-preso-a-riuso-o-di-terzi.html$ /allegato-d-guida-alla-modifica-di-software-open-source-preso-a-riuso-o-di-terzi.html permanent;
```
Comment (pdavide) [inline roles/app/templates/nginx/snippets/docs-redirects.conf]:
```suggestion
rewrite ^/italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/(.+)/attachments/allegato-e-guida-alla-modifica-di-software-open-source-preso-a-riuso-o-di-terzi.html$ /italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/(.+)/attachments/allegato-d-guida-alla-modifica-di-software-open-source-preso-a-riuso-o-di-terzi.html permanent;
```

File: roles/app/templates/nginx/snippets/docs-redirects.conf [hunk 5]
```diff
@@ -2,3 +2,10 @@
...
+rewrite ^/allegato-f-tabella-sinottica-degli-elementi-necessari-al-percorso-decisionale.html$ /allegato-e-tabella-sinottica-degli-elementi-necessari-al-percorso-decisionale.html permanent;
```
Comment (pdavide) [inline roles/app/templates/nginx/snippets/docs-redirects.conf]:
```suggestion
rewrite ^/italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/(.+)/attachments/allegato-f-tabella-sinottica-degli-elementi-necessari-al-percorso-decisionale.html$ /italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/$1/attachments/allegato-e-tabella-sinottica-degli-elementi-necessari-al-percorso-decisionale.html permanent; 
```

**Comment by yakky** [comment]
Let me know if you need any help with this

**Comment by libremente** [comment]
Waiting for @alranel review, then we can squash merge

**Comment by libremente** [comment]
@yakky @pdavide let's merge this!

**Comment by francescozaia** [comment]
Una volta online il nuovo documento, possiamo fare una pubblicazione online con questi nuovi script ed abilitare il redirect. @libremente facci sapere quando ci siete, grazie.

**Pull request merged**